src/components/RepeatableActionsField/RepeatableActionsField.js

```diff
@@ -59,21 +89,27 @@ export const RepeatableActionsField = memo(({
               dataOptions={dataOptions}
               placeholder={placeholder}
               disabled={disabled}
+              onChange={handleRepeatableActionChange}
+              validate={[validateRepeatableActions]}
             />
           )}
         </WithTranslation>
       </div>
-      {children}
+      {children(isRepeatabeFieldDisabled)}
     </div>
   );
 });
 
 RepeatableActionsField.propTypes = {
-  children: PropTypes.node.isRequired,
+  children: PropTypes.func.isRequired,
   wrapperFieldName: PropTypes.string.isRequired,
   disabled: PropTypes.bool,
   legend: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
   wrapperPlaceholder: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  repeatableFieldAction: PropTypes.string.isRequired,
```

Comment:
I guess `oneOf` PropType is more suitable here

Reply:
discussed personally, leaved as is